mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java

```diff
@@ -83,7 +83,7 @@ public MailboxException getUnderlying() {
         }
     }
 
-    private static final ModSeq FIRST_MODSEQ = new ModSeq(0);
+    private static final ModSeq FIRST_MODSEQ = ModSeq.MIN_VALUE;
```

Comment:
what about removing this constant and adding a static ModSeq.firstValue() instead?